packages/axelar-std/src/address.rs

```diff
@@ -1,9 +1,11 @@
-use soroban_sdk::{Address, Env, String};
+use soroban_sdk::{Address, Bytes, Env, String};
 
 const ZERO_ADDRESS: &str = "GAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWHF";
+const STELLAR_ADDRESS_LEN: usize = 56;
 
 pub trait AddressExt {
     fn zero(env: &Env) -> Address;
+    fn to_bytes(&self, env: &Env) -> Bytes;
```

Comment:
```suggestion

    /// add a docstring
    fn to_string_bytes(&self, env: &Env) -> Bytes;
```
rename to be consistent with the from version